backend/prisma/schema.prisma

```diff
@@ -29,13 +29,27 @@ model Business {
   location          String
   contactInformation String
   businessHours     String
-  servicesOffered   String
-  averageRating     Float     @default(0.0)
+  services          BusinessService[]
+  averageRating     Float?
   businessType      String
   reviews           Review[]
   recommendations   Recommendation[]
 }
 
+model Service {
+  id                Int       @id @default(autoincrement())
+  name              String    @unique
```

Comment:
Note that if the `name` is unique, we could instead use that as the primary key (which i assume is denoted by `@id`). This would also save having an index for the `name`, which uses more memory (`migrations.sql` line 27).
But having an `id` column is probably best since it leaves the option open to make `name` non-unique in the future I suppose.